lldb/include/lldb/Utility/Stream.h

```diff
@@ -231,6 +231,18 @@ class Stream {
   ///     The string to be output to the stream.
   size_t PutCString(llvm::StringRef cstr);
 
+  /// Output a C string to the stream with color highlighting.
+  ///
+  /// Print a C string \a text to the stream, applying color highlighting to
+  /// the specified \a pattern within the string.
+  ///
+  /// \param[in] text
+  ///     The string to be output to the stream.
+  ///
+  /// \param[in] pattern
+  ///     The portion of the \a text string to be colorized for highlighting.
```

Comment:
It should be noted that this is:
* A regex pattern that is matched as many times as possible throughout the string.
* If it is nullptr, then no highlighting is done.
* The highlight colour is red.

Reply:
✅